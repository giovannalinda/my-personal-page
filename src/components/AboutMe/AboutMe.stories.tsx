import { Meta, Story } from '@storybook/react'
import { AboutMe } from './AboutMe'

export default {
  title: 'Components/AboutMe',
  component: AboutMe,
} as Meta

export const Default: Story = () => <AboutMe />
