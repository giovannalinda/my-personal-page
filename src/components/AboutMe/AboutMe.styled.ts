import styled from 'styled-components'
import { theme } from '~/styles'

export const Container = styled.section`
  margin-top: 22rem;
  text-align: left;
`

export const Title = styled.h1`
  font-size: ${theme.font.sizes.title};
`

export const Occupation = styled.small`
  font-size: ${theme.font.sizes.paragraph};
`

export const Paragraph = styled.p`
  font-size: ${theme.font.sizes.paragraph};
  color: ${theme.colors.gray400};
  max-width: 68rem;
  padding-top: ${theme.space[7]};
  line-height: ${theme.font.lineHeights.large};
`

export const Link = styled.a`
  text-decoration: underline ${theme.colors.purple};
  color: ${theme.colors.white};
`
