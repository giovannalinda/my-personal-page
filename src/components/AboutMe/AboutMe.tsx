import * as S from './AboutMe.styled'

export function AboutMe() {
  return (
    <S.Container>
      <S.Title>
        Giovanna Souza<span>.</span>
      </S.Title>
      <S.Occupation>
        Front-end Developer na {''}
        <S.Link href='https://www.instagram.com/hypesoftbrasil/'>
          Hypesoft
        </S.Link>
      </S.Occupation>
      <S.Paragraph>
        Ol치! Prazer 游눞 <br />
        <br />
        Sou uma desenvolvedora com brilho no olho, que est치 sempre em busca do
        pr칩ximo n칤vel e que se empenha em trazer assuntos relevantes pra
        comunidade.
        <br />
        <br />
        Sou apaixonada por criar produtos 칰teis e de boa qualidade para os
        usu치rios, atualmente possuo 2 anos na 치rea de desenvolvimento e ando me
        aperfei칞oando tanto na parte t칠cnica quanto na parte de lideran칞a de um
        time.
        <br />
        <br />
        Alguns dos meus objetivos s칚o ter um crescimento continuo, impactar a
        vida das pessoas e construir interfaces de boa qualidade.
      </S.Paragraph>
    </S.Container>
  )
}
