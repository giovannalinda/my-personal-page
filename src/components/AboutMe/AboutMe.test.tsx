import { render } from '~/__helpers__/app-tests'
import { AboutMe } from './AboutMe'

describe('AboutMe', () => {
  it('matches snapshot', () => {
    const { container } = render(<AboutMe />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
