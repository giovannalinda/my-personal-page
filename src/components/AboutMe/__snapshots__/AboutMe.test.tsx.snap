// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutMe matches snapshot 1`] = `
<section
  class="sc-bczRLJ RcMkp"
>
  <h1
    class="sc-gsnTZi bRGGsb"
  >
    Giovanna Souza
    <span>
      .
    </span>
  </h1>
  <small
    class="sc-dkzDqf eZUUTv"
  >
    Front-end Developer na 
    <a
      class="sc-eCYdqJ cpJoyG"
      href="https://www.instagram.com/hypesoftbrasil/"
    >
      Hypesoft
    </a>
  </small>
  <p
    class="sc-hKMtZM bkfbAB"
  >
    Ol치! Prazer 游눞 
    <br />
    <br />
    Sou uma desenvolvedora com brilho no olho, que est치 sempre em busca do pr칩ximo n칤vel e que se empenha em trazer assuntos relevantes pra comunidade.
    <br />
    <br />
    Sou apaixonada por criar produtos 칰teis e de boa qualidade para os usu치rios, atualmente possuo 2 anos na 치rea de desenvolvimento e ando me aperfei칞oando tanto na parte t칠cnica quanto na parte de lideran칞a de um time.
    <br />
    <br />
    Alguns dos meus objetivos s칚o ter um crescimento continuo, impactar a vida das pessoas e construir interfaces de boa qualidade.
  </p>
</section>
`;
